consensus/state.go

```diff
@@ -1386,9 +1385,7 @@ func (cs *ConsensusState) recordMetrics(height int64, block *types.Block) {
 		)
 	}
 
-	cs.metrics.NumTxs.Set(float64(block.NumTxs))
```

Comment:
Why remove the metric?

Reply:
`cs.metrics.TotalTxs.Set(float64(len(block.Data.Txs)))` added this section in, NumTxs and totalTxs aren't present in the block anymore